backend/tests/test_user.py

```diff
@@ -111,3 +111,41 @@ def test_update_a_user(self):
             data=json.dumps(new_fields),
         )
         self.assertEqual(response.status_code, 404)
+
+    def test_delete_a_user(self):
+        curr_datetime = datetime.now()
+        m = MealDeliveryTask(
+            address="Test",
+            date=curr_datetime,
+            time=curr_datetime,
+            is_complete=False,
+            quantity=1,
+            type="test",
+        )
+        d = datetime.now().isoformat()
+        id = uuid.uuid4()
+        v = Volunteer(
+            id=id,
+            name="volunteer",
+            phone_number="123456789",
+            email_address="volunteer@gmail.com",
+            username="volunteer",
+            start_date=d,
+            meal_delivery_tasks=[m],
+        )
+
+        db.session.add(v)
+        db.session.commit()
+
+        response = self.client.delete("/user/{}".format(id))
+
+        # test response status code is correct
+        self.assertEqual(response.status_code, 200)
+
+        # test response body is correct
+        self.assertEqual(response.json["name"], "volunteer")
+        self.assertEqual(response.json["email_address"], "volunteer@gmail.com")
```

Comment:
You could just assertEqual the entire response body to a dictionary, its fine for now.